docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,83 @@
+
+## Dependency Dashboard
+
+One big feature of Renovate is the Dependency Dashboard, which is enabled by default.
+Read the [Key concepts, Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) page to learn more.
+
+## Grouped updates
+
+Renovate comes with community-provided groupings of dependencies, so it will group common dependencies out-of-the-box.
+With Dependabot you'll have to set your own [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).
+
+## Monorepo support
+
+Renovate has a [`group:monorepos`](https://docs.renovatebot.com/presets-group/#groupmonorepos) preset, that groups dependencies from monorepos.
+
+I don't know if Dependabot supports monorepos.
+
+## Supported platforms
+
+Platform means the Git-hosting site or program, for example GitHub, GitLab or Azure.
+
+Dependabot only works on GitHub.
+
+Renovate works on multiple platforms, including GitHub.
+Read the [list of Renovate platforms](https://docs.renovatebot.com/modules/platform/) to learn more.
+
+## Supported languages
+
+- [Renovate's supported managers](https://docs.renovatebot.com/modules/manager/)
+- [Dependabot's supported repositories and ecosystems](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems)
+
+## Customizability
+
+Both Renovate and Dependabot come with good default settings.
+
+Renovate gives you more control over _when_ and _how_ you'll get the updates.
+Renovate gives you tools to reduce the "update noise", read our [Noise reduction](https://docs.renovatebot.com/noise-reduction/) page to learn more.
+
+## Show changelogs
+
+Renovate and Dependabot show changelogs in their PRs.
+
+## Compatiblity score badges
+
+Dependabot shows one compatibility score badge.
+This score tells you how many other repositories have passing CI tests for the proposed update.
+Read the [GitHub Docs, Dependabot's compatibility scores](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates#about-compatibility-scores) to learn more about Dependabot's badge.
+
+Renovate has four _Merge Confidence_ badges:
+
+- **Age**: The age of the package
+- **Adoption**: The percentage of this package's users (within Renovate) which are using this release
+- **Passing**: The percentage of updates which have passing tests for this package
+- **Confidence**: The confidence level for this update
+
+Read the [Merge Confidence badges](https://docs.renovatebot.com/merge-confidence/) page to learn more.
+
+## Built-in to GitHub vs app
+
+Dependabot is built-in to GitHub, Renovate needs app installation or self-hosting.
```

Comment:
This should be expanded further for self-hosted teams. 

For Dependabot, the Github Enterprise license includes it for free

To add Renovate to a corporate environment there is a license cost associated as well as a separate infra needed

Reply:
Additionally, the "Dependabot alerts" included in Github are highly connected to Dependabot bot

Reply:
> Additionally, the "Dependabot alerts" included in Github are highly connected to Dependabot bot

You can select what parts of Dependabot you want to use in the repository settings. Renovate uses "Dependabot alerts" and it's "Dependency Graph" for Renovate's [`vulnerabilityAlerts`](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) feature, but the Renovate users _disables_ the normal Dependabot upgrades.

Reply:
> This should be expanded further for self-hosted executions in corporate environments.
> 
> For Dependabot, the Github Enterprise license includes it for free

Enterprises can use Renovate for free as well on GitHub. Quote from the Renovate GitHubapp installation page: [^1]

> ### Mend Free Plan
>
> Enjoy Renovate as a service for free, hosted by mend.io
>
> ✅  Private and Public repositories 

---

> To add Renovate to a corporate Github environment there is a license cost associated as well as a separate infra

If you go the self-hosting route, you'll have to pay for the server, and deal with upgrading the open source Renovate packages/container yourself. 😄 That would be no different if you decided to self-host a GitHub instance.

I think there's a possible confusion here for readers between the _paid_ Enterprise offerings of Mend, and the free opensource Renovate.

[^1]: https://github.com/marketplace/renovate

Reply:
I'm mostly staying out of this but will participate to correct factual errors like below.

> To add Renovate to a corporate Github environment there is a license cost associated

Incorrect

> as well as a separate infra

Incorrect: you can run Renovate using GitHub Actions which means no separate infra

> Additionally, the "Dependabot alerts" included in Github are highly connected to Dependabot bot

This too vague to be included in any comparison. Stick to specifics

Reply:
I stand corrected. I was under the impression that to run Renovate in a corporate environment, a license was needed

A few point stand however as running using Github Actions still involves having GitHub Actions runners (therefore, dedicated infra)

Reply:
> That would be no different if you decided to self-host a GitHub instance.

That's correct. But there is no separate _maintenance needed explicitly for dependabot_

Reply:
> Under what situations a license is needed if at all? Can this be expanded?

Renovate is Open Source. The license is right here in the project. There's not much expansion necessary beyond "Renovate is Open Source licensed"

> A few point stand however as running using Github Actions still involves having GitHub Actions runners (therefore, dedicated infra). Although it is fair to say that Github Action runners would still be needed for other workflows anyways

This is also an invalid/inaccurate point.

<img width="708" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/b3b71abc-ca3a-4c64-9b86-a24c39a38b66">

Reply:
> > That would be no different if you decided to self-host a GitHub instance.
> 
> That's correct. But there is no separate _maintenance needed explicitly for dependabot_. Github Enterprise upgrades include dependabot bundled

It's not correct to say there's no separate maintenance. There's multiple pages on managing Dependabot for GHES in addition to any regular setup for GHES + Actions:
- https://docs.github.com/en/enterprise-server@3.7/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise
- https://docs.github.com/en/enterprise-server@3.7/admin/github-actions/enabling-github-actions-for-github-enterprise-server/managing-self-hosted-runners-for-dependabot-updates

I actually think it would be faster to get a Renovate bot GitHub Action running on GitHub Enterprise Server than to follow those steps describe in the above links, but I'm not proposing to add that argument to comparison we have here. Renovate also doesn't require GitHub Actions to work with GHES either, although I'm not sure what % of enterprise self-hosters don't have Actions by now, so it may be a moot point.

Reply:
With "separate maintenance" I meant that there is no separate maintenance needed to the average user as that will be managed by the instance admins

But fair enough, it is not worth getting into that

Reply:
I finally got some clarity to explain where my confusion came from. The reason why many organizations consider Renovate to be a paid product before it can be added safely to an organization is due to its licensing  which is highlighted(intentionally or unintentionally)  when we see that the Mend Renovate On-Premises [uses a difference license model](https://docs.renovatebot.com/getting-started/running/#mend-renovate-on-premises). 

Google and many other organizations ban explicitly the usage of [AGPL](https://opensource.google/documentation/reference/using/agpl-policy)

I am not a lawyer or qualified to discuss if the reasons are valid or not, but that's where my misunderstanding came from. I am sorry for the confusion and the misleading statements corrected above. 

Therefore, Renovate is indeed open source and usable in a corporate environment, but the license might not be allowed everywhere. Any developer planning to use Renovate behind a corporate environment needs to crosscheck that the license is allowed on their organizations and mentioning the license on this comparison (As well as other places) is more than enough

Thanks!

Reply:
@yeikel you seem still wildly confused

> The reason why many organizations consider Renovate to be a paid product

You're the only person I can remember who's tried to claim that Renovate isn't free, and I don't think you have any reason to claim to speak for "many organizations".

You then justify that by mentioning Renovate On-Prem and its "license model", even though Renovate On-Prem has been available _for free_ under a very standard Enterprise-friendly EULA since 2019. 

And just to provide some grounding, your current employer is among the companies who have in the past requested one of the _free_ Renovate On-Prem licenses. Meanwhile Google, who you mention in a negative light, authorized themselves to be listed as a Renovate App reference customer on https://github.com/marketplace/renovate

Reply:
I am not a lawyer and perhaps I should have said that AGPL is not allowed _in some organizations_ with Google being an example. The Renovate source code discussed in this repo is AGPL and I do not believe that saying that is misleading

Other than that, I was unaware of the process to get a free key for the self-hosted version and what rights and limitations (if any) it gives. The [request form](https://www.mend.io/free-developer-tools/renovate/on-premises/) says that it is free but asks for a lot of personal information and to me that gives the impression that it might be the start of a sale. If that's not the case, I stand corrected as well

Lastly, and to clarify, I do not speak in name of my employer nor I know what business they have with you. My opinions and understanding (even if misled) are my own

Thanks for clarifying!